c/driver/sqlite/statement_reader.c

```diff
@@ -139,21 +139,22 @@ static AdbcStatusCode ArrowDate32ToIsoString(int32_t value, char** buf,
   time_t time = value * SECONDS_PER_DAY;
 #endif
 
+  struct tm* tmPtr;
   struct tm broken_down_time;
 
-#if defined(_WIN32)
-  if (gmtime_s(&broken_down_time, &time) != 0) {
```

Comment:
We don't need this for Windows anymore?

Reply:
We could although we aren't really even using gmtime_r here - I just copied the macro from the SQLite source to stay consistent. 

If we want to support gmtime_s the macro would have to be more complex. Windows has a gmtime_s as does C11, but the windows one reverses the order of argument. 

I think easier just to avoid that complexity and follow the sqlite pattern for now